.pre-commit-config.yaml

```diff
@@ -43,7 +43,7 @@ repos:
     hooks:
       - id: docformatter
         args: ["--in-place", "--wrap-descriptions", "79"]
-  - repo: https://github.com/open-mmlab/pre-commit-hooks
+  - repo: https://github.com/Pentahill/pre-commit-hooks
```

Comment:
```suggestion
  - repo: https://github.com/open-mmlab/pre-commit-hooks
```

Reply:
We will merge the PR (https://github.com/open-mmlab/pre-commit-hooks/pull/3) today.